src/classes/author.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+# Represents an author of items.
+class Author
+  attr_reader :id, :first_name, :last_name
+  attr_accessor :items
+
+  def initialize(id, first_name, last_name)
+    @id = id
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
```

Comment:
Same as game.rb suggestions
```suggestion
    def initialize(first_name, last_name)
    @id = Random.rand(1...1000)
    @first_name = first_name
    @last_name = last_name
    @items = []
  end
```